htdocs/adherents/class/adherent.class.php

```diff
@@ -2316,15 +2316,15 @@ function _load_ldap_info()
 		if ($this->societe && ! empty($conf->global->LDAP_MEMBER_FIELD_COMPANY))				$info[$conf->global->LDAP_MEMBER_FIELD_COMPANY] = $this->societe;
 		if ($this->address && ! empty($conf->global->LDAP_MEMBER_FIELD_ADDRESS))				$info[$conf->global->LDAP_MEMBER_FIELD_ADDRESS] = $this->address;
 		if ($this->zip && ! empty($conf->global->LDAP_MEMBER_FIELD_ZIP))						$info[$conf->global->LDAP_MEMBER_FIELD_ZIP] = $this->zip;
-		if ($this->town && ! empty($conf->global->LDAP_MEMBER_FIELD_TOWN))					$info[$conf->global->LDAP_MEMBER_FIELD_TOWN] = $this->town;
+		if ($this->town && ! empty($conf->global->LDAP_MEMBER_FIELD_TOWN))						$info[$conf->global->LDAP_MEMBER_FIELD_TOWN] = $this->town;
```

Comment:
Can you check that dolibarr function "dol_string_nohtmltag(xxx, 0)" is not better for that than hml_entity_decode+strip_tags.
dol_string_nohtmltag was designed to solve pb when using hml_entity_decode+strip_tags (see comment of dol_string_nohtmltag method)

Reply:
@eldy 
I modified to use dol_string_nohtmltag, but I modified the function so that it could use "strip_tags" instead pattern because otherwise it doubles the line break